client/src/pages/Art/Art.tsx

```diff
@@ -1,9 +1,25 @@
-import { Wrapper } from "./Art.styles";
+import { ArtSection, SectionHeader, StyledBlurb, StyledButton, StyledCompany, StyledDetail, StyledDetailDiv, StyledTitle, Wrapper } from "./Art.styles";
 
 function Art() {
     return (
         <Wrapper>
-            Art
+            <SectionHeader>ART</SectionHeader>
+            <ArtSection>
+                <StyledTitle>TITLE</StyledTitle>
+                <StyledCompany>COMPANY</StyledCompany>
+                <StyledDetailDiv>
+                    <StyledDetail>DATE</StyledDetail>
+                    <StyledDetail>LOCATION</StyledDetail>
+                </StyledDetailDiv>
+                <StyledBlurb>
+                    Blurb detailing the art and the
+                    story behind the piece. Blurb 
+                    detailing the art and the story 
+                    behind the piece. Blurb detailing 
+                    the art and the story..
+                </StyledBlurb>
+                <StyledButton>VIEW ART</StyledButton> {/* Missing > icon, I figure &gt; isn't what is desired*/}
```

Comment:
Correct Styled-Component-Icons is what we are using for this project.
@styled-icons/material-rounded/KeyboardArrowRight should be the right icon